automatic/spotify/tools/ChocolateyInstall.ps1

```diff
@@ -12,6 +12,24 @@ $arguments          = @{
     validExitCodes  = @(0, 1641, 3010)
 }
 
+# check we can use Get-AppxPackage
+if (Get-Command 'Get-AppxPackage' -ErrorAction SilentlyContinue) {
+    # there is likely going to be two packages returned for x86 and x64.
+    # we don't care about the architecture, just the version and they will both be the same.
+    $allAppxPackages = Get-AppxPackage
+    $installedAppXPackage = @($allAppxPackages | Where-Object -Property Name -eq 'SpotifyAB.SpotifyMusic')
+    if ($installedAppXPackage.Count -gt 0) {
+        if ($env:ChocolateyForce) {
+            #when you remove a package, you don't remove it per architecture. You just remove it for all architectures.
+            Write-Warning 'Attempting to remove Spotify installed from the Microsoft Store.'
+            Remove-AppxPackage -Package $installedAppXPackage[0].PackageFullName
```

Comment:
IMO, we should not remove the Microsoft Store application in this package.

It is still two different applications, which goes against having one software for each package.

IMO, I think we should throw an exception instead, no matter what, possibly with details on how the user can uninstall the application manually or add a new package that only has the purpose of removing the Store version of Spotify (that we possibly can depend on in this package).

Reply:
A package parameter? Warn them but if they add the package parameter it also uninstalls. Seems like the best of both worlds.

Reply:
hmm. Add it behind a package parameter 🤔.

Yeah, could work as well. Let's go with that instead. I still do think a separate package could be beneficial though, but we can go with the parameter for now.